src/actor-context-api.lisp

```diff
@@ -18,7 +18,8 @@
 (in-package :sento.actor-context)
 
 (defgeneric actor-of (context
-                      &key receive init destroy dispatcher state type name other-args)
+                      &key receive init destroy dispatcher state type name queue-size
```

Comment:
Would it be OK for you to not have the `queue-size` here and instead process it as part of `rest`? Because I think that to 95% of the use cases the default unbounded queue is being used.

Reply:
It will work, but I think it is much more convenient to have known arguments listed explicitly, because IDE will suggest their names.

Reply:
That is true. But I don't want users to be confused what to choose here and whether it is relevant on constructing an actor.